action/move.php

```diff
@@ -185,8 +185,8 @@ protected function updateColumnLookup(Schema $schema, Column $col, $old, $new) {
         $tconf = $col->getType()->getConfig();
         $ref = new Schema($tconf['schema']);
         if(!$ref->getId()) return; // this schema does not exist
-        if($ref->isLookup()) return; // a lookup is referenced, nothing to do
-
+        if(!$ref->getTimeStamp()) return; // a lookup is referenced, nothing to do
+        // FIXME does this make sense at all? it's always a lookup, isn't it?
```

Comment:
No, counter-intuitively you can have a Lookup-Field pointing to a page schema. Basically it's a lookup of a certain property of pages. The pid will be saved in the lookup field and needs to be updated when the page moves.